src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudRequestBuilding.scala

```diff
@@ -31,4 +31,13 @@ trait FireCloudRequestBuilding extends spray.httpx.RequestBuilding {
     addCredentials(OAuth2BearerToken("mF_9.B5f-4.1JqM"))
   }
 
+  def dummyUserIdHeaders(userId: String) = {
+    addCredentials(OAuth2BearerToken("mF_9.B5f-4.1JqM")) ~>
+      addHeader(RawHeader("OIDC_CLAIM_sub", userId)) ~>
```

Comment:
remove the _sub. I believe we've determined that the openid-connect proxy creates _sub, while the oauth proxy creates _user_id. Let's use just _user_id to mimic the current proxy.